packages/blockchain-link-utils/src/solana.ts

```diff
@@ -1,11 +1,13 @@
 import { A, D, F, pipe } from '@mobily/ts-belt';
 
 import { Target, TokenTransfer, Transaction } from '@trezor/blockchain-link-types/src';
-import type { StakeType, TokenInfo, TokenStandard } from '@trezor/blockchain-link-types/src';
-import {
-    SolanaTokenAccountInfo,
+import type {
+    StakeType,
     TokenDetailByMint,
-} from '@trezor/blockchain-link-types/src/solana';
+    TokenInfo,
+    TokenStandard,
+} from '@trezor/blockchain-link-types/src';
+import { SolanaTokenAccountInfo } from '@trezor/blockchain-link-types/src/solana';
```

Comment:
nit: (here and other places) maybe export those solana things from @trezor/blockchain-link-types and
```suggestion
import { SolanaTokenAccountInfo } from '@trezor/blockchain-link-types';
```

wdyt?